dev/reference/tools/data-migration/manage-tasks.md

```diff
@@ -464,50 +449,377 @@ Flags:
   -h, --help   help for update-task
 
 Global Flags:
-  -w, --worker strings   dm-worker ID
+  -w, --worker strings   DM-worker ID
 ```
 
 #### Command usage example
 
 ```bash
-update-task [-w "127.0.0.1:10181"] ./task.yaml
+update-task [-w "127.0.0.1:8262"] ./task.yaml
 ```
 
 #### Flags description
 
-- `-w`: (Optional) This flag specifies the group of DM-workers where the subtasks of the replication task (that you want to update) run. If it is set, only subtasks on the specified DM-workers are updated.
-- `config_file`: (Required) This flag specifies the file path of `task.yaml`.
+- `-w`: (Optional) Specifies the group of DM-workers where the subtasks of the replication task (that you want to update) run. If it is set, only subtasks on the specified DM-workers are updated.
+- `config-file`: (Required) Specifies the file path of `task.yaml`.
 
 #### Returned results
 
 ```bash
-» update-task task_all_black.yaml
+update-task task_all_black.yaml
+```
+
+```bash
 {
 ​    "result": true,
 ​    "msg": "",
 ​    "workers": [
 ​        {
 ​            "result": true,
-​            "worker": "172.16.30.15:10081",
+​            "worker": "172.16.30.15:8262",
 ​            "msg": ""
 ​        },
 ​        {
 ​            "result": true,
-​            "worker": "172.16.30.16:10081",
+​            "worker": "172.16.30.16:8262",
 ​            "msg": ""
 ​        }
 ​    ]
 }
 ```
 
-## Manage the DDL locks
+## Manage DDL locks
+
+Currently, DDL lock related commands mainly include `show-ddl-locks`, `unlock-ddl-lock`, `break-ddl-lock`, etc. For more information on their functions, usages, and applicable scenarios, refer to [Handle Sharding DDL Locks Manually](/dev/reference/tools/data-migration/features/manually-handling-sharding-ddl-locks.md).
+
+## Other task and cluster management commands
+
+In addition to the common task management commands above, DM also provides some other commands to manage data replication tasks and DM clusters.
+
+### Check the task configuration file
+
+You can use the `check-task` command to check whether a specified configuration file (`task.yaml`) of the replication task is valid, or whether the configuration of upstream/downstream database, permission setting, and schema meet the replication requirements. For more details, refer to [Precheck the upstream MySQL instance configuration](/dev/reference/tools/data-migration/precheck.md).
+
+When you use `start-task` to start a replication task, DM also executes all checks done by `check-task`.
+
+{{< copyable "" >}}
+
+```bash
+help check-task
+```
+
+```
+check the config file of the task
+
+Usage:
+ dmctl check-task <config-file> [flags]
```

Comment:
you can change  all `config_file` to `config-file` in this file.

Reply:
@csuzhangxc Currently, both `config_file` and `config-file` exist in [the Chinese version](https://pingcap.com/docs-cn/v3.0/reference/tools/data-migration/manage-tasks/). Do we need to modify all `config_file` to `config-file` to keep them consistent? Otherwise, I'm afraid users might feel confused.

Reply:
> Do we need to modify all config_file to config-file to keep them consistent?

YES

Reply:
@csuzhangxc Got it. @TomShawn Please help modify all `config_file` in [the Chinese version](https://pingcap.com/docs-cn/v3.0/reference/tools/data-migration/manage-tasks/) to `config-file` (for three versions). Thanks!